### ISSUE TITLE
dx download/upload fails on FIPS enabled systems

### ISSUE THREAD

**Issue by anastasia-lucas** [dnanexus/dx-toolkit#702]
Hello, would it be possible to add support for FIPS enabled systems? 

Currently at least `dx download` and `dx upload` are failing with the following error

`ValueError: [digital envelope routines: EVP_DigestInit_ex] disabled for fips`

It seems like python 3.9 has built in support for this and it looks like this link has a similar example of an md5 on FIPS enabled system.
https://github.com/sphinx-doc/sphinx/issues/7611#issue-611746028

**Comment by kpjensen** [comment]
Hello, I opened a branch to try and support this. If you install dxpy via pip, you can try installing this branch like so. 

```
python3 -m pip install -U "git+https://github.com/dnanexus/dx-toolkit.git@DEVEX-1961-md5#egg=dxpy&subdirectory=src/python"
```

**Comment by anastasia-lucas** [comment]
@kpjensen this is working for me. Thanks so much for the quick response!

**Comment by kpjensen** [comment]
Closing as this is now released in `0.314.0`
